test/api/v3/integration/tasks/DELETE-tasks_id.test.js

```diff
@@ -16,7 +16,7 @@ describe('DELETE /tasks/:id', () => {
     let task;
 
     beforeEach(() => {
-      return user.post('/tasks', {
+      return user.post('/tasks?tasksOwner=user', {
```

Comment:
Okay, I'll try to write a regex that matches /task/(user/challenge)/optional challenge id without matching /tasks/taskId. If you've more experience with regex feel free to do it, I'm not very good at them